src/wp-includes/option.php

```diff
@@ -277,6 +277,19 @@ function form_option( $option ) {
 function wp_load_alloptions( $force_cache = false ) {
 	global $wpdb;
 
+	/**
+	 * This filter allows the wp load all options function to be shortcut.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param null|array $pre if an array is returned then the wp_load_alloptions will be shortcuted and the array returned.
+	 * @param bool       $force_cache Whether to force an update of the local cache from the persistent cache. Default false.
```

Comment:
```suggestion
	 * Filters the array of alloptions before it is populated.
	 *
	 * Returning a non-null value from the filter will effectively short circuit
	 * wp_load_alloptions(), returning that value instead.
	 *
	 * @since 6.2.0
	 *
	 * @param null|array $alloptions  An array of alloptions.
	 * @param bool       $force_cache Whether to force an update of the local cache from the persistent cache. Default false.
```

Reply:
Renaming the parameter to be a bit more descriptive.

Reply:
Also tried to emulate docblocks in similar filters.